extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -727,17 +663,6 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
   companion object {
     private const val TAG = "GeoJsonSource"
 
-    /**
-     * Check if runtime property geojson_allow_direct_setter is enabled.
-     */
-    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    fun directSetterEnabled(): Boolean {
```

Comment:
we could have this as internal, but ok to remove them as we had the VisibleForTesting annotation.

Reply:
not sure that understand correctly. we don't need this `directSetterEnabled` at all. direct setting will be default behaviour

Reply:
I mean this API is public visible in v10.10.0 release, it could have been made as internal, otherwise, it's theoretically a breaking change, as the api file indicates.
However, I'm ok with removing it here in this PR, it shouldn't be used anyway.

Reply:
well, it's appears only for that feature, and now it's not necessary any more. I guess we can remove it. 
and yes, it will be a breaking change is someone used it